### DISCUSSION TITLE
app launch screen

### DISCUSSION THREAD

**Discussion by massimilianocom** [apache/cordova#473] [General]
I created a small app and everything works great but I don't understand one thing, that is, when I start the app on the smartphone while the app is loading I get a white screen with the Cordova logo in the center, how can I get rid of this screen?

**Comment by breautek** [comment]
Need to add in your splashscreen resources. See https://cordova.apache.org/docs/en/12.x/core/features/splashscreen/index.html for more details.

**Comment by massimilianocom** [comment]
Following the guide you kindly posted, I inserted this into the main directory config.xml file:

`        <preference name="AndroidWindowSplashScreenBackground" value="#ffffff" />
        <preference name="AndroidWindowSplashScreenAnimatedIcon" value="res/android/res/android/xxxhdpi.png" />
`

but nothing doesn't work

Still using the guide, I also tried to delete the screen with this:

`document.addEventListener('deviceready', onDeviceReady, false);

function onDeviceReady() {
        navigator.splashscreen.hide();
}

Even in this case it doesn't work, the screen continues to be displayed
